Remotion/ObjectBinding/Core/BusinessObjectStringFormatterService.cs

```diff
@@ -237,15 +237,15 @@ private string GetStringValueForReferenceProperty (object value)
       if (value is IBusinessObjectWithIdentity)
         return ((IBusinessObjectWithIdentity) value).GetAccessibleDisplayName();
 
-      return value.ToString();
+      return Assertion.IsNotNull (value.ToString(), "The ToString() method of the reference property must not return null.");
```

Comment:
please add the property.Identifier in the message 
`...reference property '{0}' must...`

also, we could probably drop the `reference` part.